web-dev/Frontend/Libraries & Frameworks/Learn-React.md

```diff
@@ -41,6 +41,9 @@
   * [React without ES6](#react-without-es6)
   * [Reconcilation - Diff Algorithm](#reconcilation---diff-algorithm)
 
+* [Complete Projects](#complete-projects-that-you-can-have-a-look)
+* [Opensource projects you can contribute to](#opensource-projects-you-can-contribute-to)
```

Comment:
Did you generate table of content with `doc-toc` or did you insert manually?

Reply:
Manually

Reply:
Shall we add doc-toc as a git hook?
https://github.com/thlorenz/doctoc#usage-as-a-git-hook

Reply:
yes please add